plugins/snap.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: snap
+spec:
+  version: "v0.0.1"
+  homepage: https://github.com/honk-ci/kubectl-snap
+  platforms:
+  - selector:
+      matchExpressions:
+      - {key: os, operator: In, values: [darwin, linux]}
+    uri: https://github.com/honk-ci/kubectl-snap/releases/download/0.1/kubectl-snap-0.1.tar.gz
+    sha256: "8df10f76e82f71b34c737d128b7b369e0daf641118577822872537e787a6755d"
+    files:
+    - from: "*"
+      to: .
+    bin: kubectl-snap
+  shortDescription: A chaos test that restores balance to your cluster at the speed of a snap.
```

Comment:
Doesn't explain what the plugin that.
`description` field does a better job.

Let's just say `Randomly delete pods in namespace or cluster for chaos testing`

Reply:
Fair fair ;) Updating.